recipes/gst-plugins-base/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,44 @@
+#include <gst/gst.h>
```

Comment:
I think this source can be easily converted to c.
It would also test whether the cppstd library is linked correctly, if needed by a dependency

Reply:
it can be converted, but it needs some minor modifications (e.g. replace std::iostream with printf)